lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -188,11 +190,21 @@ public APIGatewayProxyResponseEvent handleRequest(
                     ipvSessionService.generateIpvSession(
                             clientOAuthSessionId, null, emailAddress, isReverification);
 
+            Optional<ListOfStringValues> evcsAccessTokenClaim = Optional.empty();
+            if (configService.enabled(EVCS_READ_ENABLED)) {
```

Comment:
This probably needs to be reads _or_ writes, given that we need a read when storing identity?

Reply:
checked both flags